modules/react-components/src/components/accordion/segmented-accordion/segmented-accordion-title.tsx

```diff
@@ -119,7 +119,7 @@ export interface StrictSegmentedAccordionTitleActionInterface {
  *
  * @param props - Props injected to the component.
  *
- * @returns Segmented Accordion Title React Component
+ * @returns
```

Comment:
Can we add what we are returning?

Reply:
resolved with https://github.com/wso2/identity-apps/pull/3528/commits/acf93bbed6a21c13a9d22d84cc8294ea664ad8c4